csrc/device_lower/analysis/index_compute.cpp

```diff
@@ -838,7 +838,7 @@ IndexFromIdGraph getTensorIndexFromIdGraph(
     const TensorView* consumer_tv,
     const TensorView* producer_tv,
     bool is_global,
-    std::unordered_map<IterDomain*, IterDomain*> c2p_map) {
+    const std::unordered_map<IterDomain*, IterDomain*> c2p_map) {
```

Comment:
Unrelated to this PR, but clang-tidy asked me to fix some const-correctness, so I fixed some nearby missing const while I was at it.

Reply:
Should this be `const std::unordered_map<IterDomain*, IterDomain*>&` instead of `const std::unordered_map<IterDomain*, IterDomain*>`?

Reply:
Nice catch. I wonder why `clang-tidy` doesn't find that??